src/content/docs/en/guides/images.mdx

```diff
@@ -488,18 +488,7 @@ The [Astro template syntax](/en/reference/astro-syntax/) also supports writing a
 
 ### images in `src/`
 
-Local images must be imported from the relative path from the existing `.astro` file, or you can configure and use an [import alias](/en/guides/imports/#aliases). Then, you can access the image's `src` and other properties to use in the `<img>` tag.
-
-Imported image assets match the following signature:
-
-```ts
-interface ImageMetadata {
-  src: string;
-  width: number;
-  height: number;
-  format: string;
-}
-```
+Local images must be imported from the relative path from the existing `.astro` file, or you can configure and use an [import alias](/en/guides/imports/#aliases). They become available as an [`ImageMetadata` object](/en/reference/modules/astro-assets/#imagemetadata-1) giving you access to the image's `src` and other properties to use in the `<img>` tag.
```

Comment:
I like link update, but I do wonder if keeping example type is helpful here (even if content is duplicated). One of the issues we have in explaining how images work is that if you're just using the HTML image tag, you have to rely on the object for its properties (e.g. `image.src`) vs if you use the components, you can just write `src`). The extra reinforcement here might be good, actually, and not force people to click away for something that is kind of short anyway?

What do you think?